egicli/endpoint.py

```diff
@@ -0,0 +1,294 @@
+from __future__ import print_function
+
+import os
+import json
+import xml.etree.ElementTree as ET
+
+import click
+from six.moves.urllib import parse
+import requests
+from tabulate import tabulate
+
+from egicli.checkin import refresh_access_token
+
+GOCDB_PUBLICURL = "https://goc.egi.eu/gocdbpi/public/"
+
+
+def find_endpoint(service_type, production=True, monitored=True, site=None):
+    q = {"method": "get_service_endpoint", "service_type": service_type}
+    if monitored:
+        q["monitored"] = "Y"
+    if site:
+        q["sitename"] = site
+    url = "?".join([GOCDB_PUBLICURL, parse.urlencode(q)])
+    # XXX UGLY, but GOC is using a IGTF CA not in standard distros
```

Comment:
Doest it refer to disabling certificate validation that was removed?

Reply:
yes, no longer applies